resources/startup.sh

```diff
@@ -34,33 +46,60 @@ TEMPORARY_ADMIN_GROUP=$(doguctl random)
 TEMPORARY_ADMIN_USER=$(doguctl random)
 TEMPORARY_ADMIN_PASSWORD=$(doguctl random)
 
-function import_quality_profiles_if_present {
-  AUTH_USER=$1
-  AUTH_PASSWORD=$2
+function areQualityProfilesPresent() {
   echo "Check for quality profiles to import..."
-  # only import profiles if quality profiles are present
-  if [[ "$(ls -A "${QUALITY_PROFILE_DIR}")" ]];
-  then
-    # import all quality profiles that are in the suitable directory
-    for file in "${QUALITY_PROFILE_DIR}"/*
-    do
-      # ignore CasAuthenticationExceptions in log file, because the credentials below only work locally
-      RESPONSE_IMPORT=$(curl ${CURL_LOG_LEVEL} --fail -X POST -u "${AUTH_USER}":"${AUTH_PASSWORD}" -F "backup=@$file" localhost:9000/sonar/api/qualityprofiles/restore)
-      # check if import is successful
-      if ! ( echo "${RESPONSE_IMPORT}" | grep -o errors);
-      then
-        echo "Import of quality profile $file was successful"
-        # delete file if import was successful
-        rm -f "$file"
-        echo "Removed $file file"
-      else
-        echo "Import of quality profile $file has not been successful"
-        echo "${RESPONSE_IMPORT}"
-      fi;
-    done;
+
+  local resultCode
+  if [[ "$(ls -A ${QUALITY_PROFILE_DIR})" ]]; then
+    echo "Quality profiles are present..."
```

Comment:
This statement does only apply to the quality profile directory and not to SonarQube in general, so it may be misleading in the log.
```suggestion
    echo "Quality profiles are present for import..."
```

Reply:
Hmmm, this is a code duplication with that from above (line 57). But point is taken